Bottom: f645979e703e3337aa2e7456b7e1742bc602ce6a
Top:    145c31a6954c193a2ed1d3716d0fb4db0b11cd14
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-10-09 09:55:37 -0700

Refresh of 1-5-librdmacm-rsocket-add-miss

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 278c8ec..5b65b12 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -2002,8 +2002,8 @@ int rfcntl(int socket, int cmd, ... /* arg */ )
 	va_start(args, cmd);
 	switch (cmd) {
 	case F_GETFL:
-		va_end(args);
-		return (int) rs->fd_flags;
+		ret = (int) rs->fd_flags;
+		break;
 	case F_SETFL:
 		param = va_arg(args, long);
 		if (param & O_NONBLOCK)
@@ -2014,6 +2014,7 @@ int rfcntl(int socket, int cmd, ... /* arg */ )
 		break;
 	default:
 		ret = ERR(ENOTSUP);
+		break;
 	}
 	va_end(args);
 	return ret;
